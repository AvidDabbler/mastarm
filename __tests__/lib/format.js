/* globals afterEach, beforeEach, describe, expect, it */

const fs = require('fs')

const rimraf = require('rimraf')

const format = require('../../lib/format')

describe('format', () => {
  beforeEach(done => fs.writeFile('test.js', 'function a(){return 1;}', done))
  afterEach(done => rimraf('test.js', done))

  it('can format a file', done => {
    format(['test.js']).then(() => {
      expect(fs.readFileSync('test.js', {encoding: 'utf-8'})).toMatchSnapshot()
      done()
    })
  })
})
