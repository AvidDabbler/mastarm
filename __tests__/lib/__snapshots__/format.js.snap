// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`format can format a file 1`] = `
"function a () {
  return 1
}
"
`;
